API.md

```diff
@@ -0,0 +1,48 @@
+# AmperkaTDS API
+
+## `class AmperkaTDS`
+
+Create an object of type `AmperkaTDS` to communicate with a particular [TDS-sensor](https://amperka.ru/product/troyka-tds-sensor).
+
+### `AmperkaTDS(uint8_t pinS)`
+
+Constructs a new `AmperkaTDS` object. The argument `pinS` the analog output signal. It is marked `S` on the TDS board. Use any analog GPIO pin of the controller.
+
+### `void begin()`
+
+Initializes the given interface, prepares the board for communication.
+Call this method before interfacing with `AmperkaTDS`. For example, in your `setup()`.
+
+### `void read()`
+
+Reads and stores the sensor data values: EC and TDS.
+
+### `uint16_t getEC()`
+
+Returns the stored electrical conductivity (EC) when the call method `read`. EC measurement is in microsiemens per centimeter and ranges from `0` to `2500` in `uS/cm`.
+
+### `uint16_t getTDS()`
+
+Returns the stored total dissolved solids (TDS) when the call method `read`. TDS measurement is in parts per million and ranges from `0` to `1250` in `ppm`.
+
+### `float getCalibrationFactor()`
```

Comment:
Is it something that involves real communication with the sensor? I think a user should call this after putting the probe into a liquid with known parameters. Does he? If so, add a short description of the process and use verb _read_ in the signature:

```suggestion
### `float readCalibrationFactor()`
```

If it just returns a stored value, should be:

```suggestion
### `float getCalibrationFactor() const`
```

Reply:
@nkrkv You're right. I fixed this function and added a short description of the calibration.